include/items.php

```diff
@@ -763,6 +764,17 @@ function item_store($arr,$force_parent = false, $notify = false, $dontcache = fa
 		return 0;
 	}
 
+	// Check for already added items.
+	// There is a timing issue here that sometimes creates double postings.
+	// An unique index would help - but the limitations of MySQL (maximum size of index values) prevent this.
```

Comment:
I'm able to add a UNIQUE index on `uid` and `uri` of th `item` table. Of course the `uri` field still is in `utf8_general_ci` collation because we hopefully don't need emojis in item uri.